mesheryctl/internal/cli/root/system/check.go

```diff
@@ -240,8 +242,15 @@ func (hc *HealthChecker) runDockerHealthChecks() error {
 	if hc.Options.PrintLogs {
 		log.Info("\nDocker \n--------------")
 	}
+	// Create docker client using ENV variables
+
+	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		return err
```

Comment:
Can we add a MeshKit Error here, @Azanul?

Reply:
MeshKit Errors - https://docs.meshery.io/project/contributing/contributing-error